src/pages/blogg/[slug].tsx

```diff
@@ -35,7 +35,16 @@ const blogQuery = groq`
     tittel,
     _createdAt,
     mainImage,
-    body,
+    body[]{
```

Comment:
Jeg så noe sanity-doc som mente at dette var den beste måten å resolve `reference` i blockcontent, men tar gjerne i mot tips på bedre løsninger. :)

Også usikker på om vi må legge inn dette for andre steder vi bruker blockcontent?